Add numbers to hexagons in join section

malte-hansen opened the pull request:
This pull request closes #30 . However, the solution is, just as the other hexagons on the website, not easily configurable. This is reflected in another issue (see #43), which asks for a approach which is applicable to all hexagon elements which we use.

Also, since #30 is a design issue, I would like the requested reviewers to comment on the proposed solution. Of course, you might as well reject the pull request if you dislike the added numbers (in comparison to the previous design).

malte-hansen committed:
Add numbers to hexagons in join section

MalteRei commented:
I love the numbers. I think they give the hexagons more of a purpose. I actually also like the rounded font type pf the numbers. Event though this might be debatable if different fonts on the same page introduce to much inconsistency.

MalteRei commented:
The maintainability of the hexagons seems bad.
In addition loading 3 svg's is probably not helpful performance wise compared to loading just one svg.

May I propose a solution that should fix these issues:

Commit:
Make hexagons with numbers more maintainable

No need to create and load 3 different files if the numbers can be defined in the html.

TODO: If we decide to go with this version we can delete hexagon_1.svg, hexagon_2.svg and hexagon_3.svg

malte-hansen commented:
Thanks for the adapted solution. 
However, before merging this, @Dominik-MCC and I would like to test how icons (e.g. using the awesome font) would look for the hexagons. The numbers look okay but do not add much visual information about the necessary actions to become a member.

MalteRei commented:
> Thanks for the adapted solution.
> However, before merging this, @Dominik-MCC and I would like to test how icons (e.g. using the awesome font) would look for the hexagons. The numbers look okay but do not add much visual information about the necessary actions to become a member.

Greate idea!
Maybe use Fluent Ui Icons. Because we are already using Segoe Ui as Font and are Microsoft affiliated. 😉
https://developer.microsoft.com/en-us/fluentui#/styles/web/icons
